File: sample/java/src/main/java/io/github/simonschiller/prefiller/sample/product/ProductDatabase.java
package io.github.simonschiller.prefiller.sample.product;

import androidx.room.Database;
import androidx.room.RoomDatabase;

import io.github.simonschiller.prefiller.sample.order.Order;
import io.github.simonschiller.prefiller.sample.order.OrderDao;

@Database(entities = {Product.class, Order.class}, version = 3)
public abstract class ProductDatabase extends RoomDatabase {
    public abstract ProductDao productDao();
    public abstract OrderDao orderDao();
}

File: sample/kotlin-ksp/src/main/kotlin/io/github/simonschiller/prefiller/sample/product/ProductDatabase.kt
package io.github.simonschiller.prefiller.sample.product

import androidx.room.Database
import androidx.room.RoomDatabase
import io.github.simonschiller.prefiller.sample.order.Order
import io.github.simonschiller.prefiller.sample.order.OrderDao

@Database(entities = [Product::class, Order::class], version = 3)
abstract class ProductDatabase : RoomDatabase() {
    abstract fun productDao(): ProductDao
    abstract fun orderDao(): OrderDao
}
